crates/interpreter/src/instructions/contract/call_helpers.rs

```diff
@@ -136,51 +145,41 @@ pub fn load_account_delegated<H: Host + ?Sized>(
     let is_berlin = spec.is_enabled_in(SpecId::BERLIN);
     let is_spurious_dragon = spec.is_enabled_in(SpecId::SPURIOUS_DRAGON);
 
-    let skip_cold_load = is_berlin && remaining_gas < COLD_ACCOUNT_ACCESS_COST_ADDITIONAL;
+    let additional_cold_cost = gas_table.cold_account_additional_cost();
+
+    let skip_cold_load = is_berlin && remaining_gas < additional_cold_cost;
     let account = host.load_account_info_skip_cold_load(address, true, skip_cold_load)?;
     if is_berlin && account.is_cold {
-        cost += COLD_ACCOUNT_ACCESS_COST_ADDITIONAL;
+        cost += additional_cold_cost;
     }
     let mut bytecode = account.code.clone().unwrap_or_default();
     let mut code_hash = account.code_hash();
     // New account cost, as account is empty there is no delegated account and we can return early.
     if create_empty_account && account.is_empty {
-        cost += new_account_cost(is_spurious_dragon, transfers_value);
+        cost += gas_table.new_account_cost(is_spurious_dragon, transfers_value);
         return Ok((cost, bytecode, code_hash));
     }
 
     // load delegate code if account is EIP-7702
     if let Some(Bytecode::Eip7702(code)) = &account.code {
         // EIP-7702 is enabled after berlin hardfork.
-        cost += WARM_STORAGE_READ_COST;
+        cost += gas_table.warm_storage_read_cost();
         if cost > remaining_gas {
             return Err(LoadError::ColdLoadSkipped);
         }
         let address = code.address();
 
         // skip cold load if there is enough gas to cover the cost.
-        let skip_cold_load = remaining_gas < cost + COLD_ACCOUNT_ACCESS_COST_ADDITIONAL;
+        let skip_cold_load = remaining_gas < cost + additional_cold_cost;
         let delegate_account =
             host.load_account_info_skip_cold_load(address, true, skip_cold_load)?;
 
         if delegate_account.is_cold {
-            cost += COLD_ACCOUNT_ACCESS_COST_ADDITIONAL;
+            cost += additional_cold_cost;
         }
         bytecode = delegate_account.code.clone().unwrap_or_default();
         code_hash = delegate_account.code_hash();
     }
 
     Ok((cost, bytecode, code_hash))
 }
-
-/// Returns new account cost.
-#[inline]
-pub fn new_account_cost(is_spurious_dragon: bool, transfers_value: bool) -> u64 {
-    // EIP-161: State trie clearing (invariant-preserving alternative)
-    // Pre-Spurious Dragon: always charge for new account
-    // Post-Spurious Dragon: only charge if value is transferred
-    if !is_spurious_dragon || transfers_value {
-        return NEWACCOUNT;
-    }
-    0
```

Comment:
Moved to: https://github.com/bluealloy/revm/blob/0ee23dfbbc19f48d92e2a9faedd41f184a28aef4/crates/interpreter/src/gas/params.rs#L442-L453